dev/inc/SharedHelpers.h

```diff
@@ -309,6 +309,8 @@ class SharedHelpers
         }
     }
 
+    static winrt::float4 RgbaColor(winrt::Color color);
```

Comment:
are you able to mark this helper as constexpr or does that not compile?

Reply:
I don't think this will work since color values are not known at compile time.